pandas/tests/tools/test_concat.py

```diff
@@ -1825,6 +1825,15 @@ def test_concat_bug_3602(self):
         result = concat([df1, df2], axis=1)
         assert_frame_equal(result, expected)
 
+    def test_concat_bug_15328(self):
+        df_empty = pd.DataFrame()
```

Comment:
put a comment here for the issue number (rather than in the test name) (try to be descriptive with that if possible)